workflows/alevin-quant/run-alevin-fry.nf

```diff
@@ -0,0 +1,124 @@
+#!/usr/bin/env nextflow
+nextflow.enable.dsl=2
+
+// run parameters
+params.ref_dir = 's3://nextflow-ccdl-data/reference/homo_sapiens/ensembl-100'
+params.index_dir = 'salmon_index'
+params.index_name = 'txome_k31'
+params.annotation_dir = 'annotation'
+params.t2g = 'Homo_sapiens.ensembl.100.tx2gene.tsv'
+params.mitolist = 'Homo_sapiens.ensembl.100.mitogenes.txt'
+
+params.run_metafile = 's3://ccdl-scpca-data/sample_info/scpca-library-metadata.tsv'
+// run_ids are comma separated list to be parsed into a list of run ids,
+// or "All" to process all samples in the metadata file
+params.run_ids = "SCPCR000001,SCPCR000002"
+
+params.outdir = 's3://nextflow-ccdl-results/scpca/alevin-fry-quant'
+
+// build full paths
+params.index_path = "${params.ref_dir}/${params.index_dir}/${params.index_name}"
+params.t2g_path = "${params.ref_dir}/${params.annotation_dir}/${params.t2g}"
+params.mito_path = "${params.ref_dir}/${params.annotation_dir}/${params.mitolist}"
+
+// generates RAD file using alevin
+process alevin{
+  container 'quay.io/biocontainers/salmon:1.4.0--hf69c8f4_0'
+  label 'cpus_8'
+  tag "${id}-${index}"
+  publishDir "${params.outdir}"
+  input:
+    tuple val(id), path(read1), path(read2)
+    path index
+    path tx2gene
+  output:
+    path run_dir
+  script:
+    run_dir = "${id}-${index}"
+  // run alevin like normal with the --justAlign flag 
+  // creates output directory with RAD file needed for alevin-fry
+    """
+    mkdir -p ${run_dir}
+    salmon alevin \
+      -l ISR \
+      --chromium \
+      -1 ${read1} \
+      -2 ${read2} \
+      -i ${index} \
+      --tgMap ${tx2gene} \
+      -o ${run_dir} \
+      -p ${task.cpus} \
+      --dumpFeatures \
+      --justAlign
+    """
+}
+
+//generate permit list from RAD input 
+process generate_permit{
+  container 'ghcr.io/alexslemonade/scpca-alevin-fry:latest'
+  label 'cpus_8'
```

Comment:
Hi @jashapiro : Just to weigh in on the "resolution strategy" question.  I think it is worth testing a few.  I actually think that in the next release of `alevin-fry` we may disable any "default" strategy and focus on providing sufficient documentation for the user to make a (hopefully) informed decision.  Right now, I'd say that it's **definitely worth testing** `cr-like` if you are very concerned with evaluating the speed of the pre-processing pipelines.  The method closest to cell ranger would likely be something like `parsimony`, which does 1-edit UMI collapse, but doesn't follow up with an EM.  Actually, parsimony is already doing quite a bit more than "simple" 1-edit UMI collapse,  since it is doing 1-edit UMI collapse _and_ trying to resolve any multi-gene-spanning UMIs using the minimum cover monochromatic arborescence formulation of alevin.  The only real difference between `full` and `parsimony` is that `parsimony` discards UMIs that have equally parsimonious covers attributed to multiple genes, while full resolves them probabilistically with an EM.  We'll certainly consider adding a mode that does 1-edit UMI collapse in a manner more similar to how CellRanger does it (without the parsimony criterion) if that makes sense.  I know that STARsolo implements a collapse approach very similar to that of CellRanger.